client/src/components/SearchResult/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import sitspotImg from '../../assets/searchResult';
```

Comment:
so, you're using the index.js file of the above assets directory as sort of a map of name -> imgSrc. It is done the same way for the MagicalFactor component. I would get rid of those index.js files in the assets folder and replace them with one module that would map all of the images (let's call it `imageRepository.js`):
```js
import image1 from {path_to_assets}/{path_to_image1};
import image2 from {path_to_assets}/{path_to_image2};

export default const ImageRepository = Object.freeze({
    name1: image1,
    name2: image2,
});
```
The benefit is it would be unified across the project instead of being split into multiple index.js files inside of the assets folder and thus easier to understand and maintain (it could be even made as an API: `imageRepo.getImage(img)` instead of `imageRepo[img]`)